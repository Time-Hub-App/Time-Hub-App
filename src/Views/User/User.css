.create-journal {
  font-weight: bold;
  text-decoration: underline;
  color: #727397;
}

.create-journal:hover {
  cursor: pointer;
}
