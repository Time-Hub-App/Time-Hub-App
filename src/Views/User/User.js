import './User';
import Header from '../../Components/Header/Header.js';
import Footer from '../../Components/Footer/Footer.js';
import JournalList from '../../Components/JournalList/JournalList';
import { NavLink } from 'react-router-dom';

export default function User() {
  return (
    <div>
      <Header />
      <NavLink to="/add">Create Journal</NavLink>
      <JournalList />
      <Footer />
    </div>
  );
}
