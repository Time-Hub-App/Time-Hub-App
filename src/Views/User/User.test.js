import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import User from './User';

it('should render a user view component', () => {
  const { container } = render(
    <MemoryRouter>
      <User />
    </MemoryRouter>
  );

  expect(container).toMatchSnapshot();
});
