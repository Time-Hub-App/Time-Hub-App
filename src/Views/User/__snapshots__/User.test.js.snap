// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a user view component 1`] = `
<div>
  <div>
    <header>
      <h3
        class=""
      >
        Home
      </h3>
      <h3
        class=""
      >
        About
      </h3>
      <h3
        class=""
      >
        Log In
      </h3>
      <h3
        class="active"
      >
        Journals
      </h3>
    </header>
    <h2
      class="create-journal"
    >
      Create Journal
    </h2>
    <div
      class="journallist"
    />
    <footer>
      <a
        class="footer-link"
        href="https://github.com/Time-Hub-App"
      >
        TimeHub GitHub
      </a>
    </footer>
  </div>
</div>
`;
